### DISCUSSION TITLE
Documentation ?

### DISCUSSION THREAD

**Discussion by kunalkarda** [rwightman/efficientdet-pytorch#196] [General] [answered]
Where can we find the documentation to completely understand the library and use it efficiently ?
Thank you

**Comment by rwightman** [comment, marked as answer]
@kunalkarda you can pay me my hourly rate to write some, or you can refer to @amaarora excellent write (please thank him for the work) https://amaarora.github.io/2021/01/13/efficientdet-pytorch.html
